### ISSUE TITLE
Update prints to use a consistent format:

### ISSUE THREAD

**Issue by KennethEnevoldsen** [embeddings-benchmark/mteb#1046]
We seem to use different methods for print throughout the code base, e.g.:
```
----------Using 8 data-parallel GPUs----------
─────────────────────────────── Selected tasks  ────────────────────────────────
```

We should probably be consistent in these

**Comment by isaac-chung** [comment]
Might want to standardize logging as well - could be a separate issue.

**Comment by KennethEnevoldsen** [comment]
So whether we using the logging module, console.print (rich)

**Comment by KennethEnevoldsen** [comment]
I think there is still a need for this change, but primarily in the `mteb.MTEB` and the remainder of the code.

**Comment by ayush1298** [comment]
> I think there is still a need for this change, but primarily in the `mteb.MTEB` and the remainder of the code.
>
Should I take this now, or once after v2 is merged?

**Comment by KennethEnevoldsen** [comment]
don't take it know
